src/test/java/com/afornalik/ox/TestMatch.java

```diff
@@ -13,58 +9,41 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @Test
-public class TestMatchController {
+public class TestMatch {
 
-    private static final Board BOARD = new Board(5);
-    private static final Board BOARD_THREE = new Board(3);
+    private static final Board BOARD_THREE = new Board(3, 3);
     private static final PlayerContainer PLAYER_CONTAINER = new PlayerContainer();
 
-    public void shouldUserMakeAMove() throws OutOfBoardException {
-        //given
-        UIExtended uiConsole = Mockito.mock(UIExtended.class);
-        Mockito.when(uiConsole.readNumber()).thenReturn(1);
-        MatchController matchController = new MatchController(uiConsole, new Board(5), PLAYER_CONTAINER);
-
-        //when
-        Board result = matchController.doMove(FieldStatus.X);
-        BOARD.insertBoardField(1, FieldStatus.X);
-
-        //then
-        assertThat(result.receiveBoardField(0)).isEqualTo(FieldStatus.X);
-        assertThat(result.receiveBoardField(0)).isEqualTo(BOARD.receiveBoardField(1));
-    }
-
-
     @Test
     public void shouldAllFieldBeMarked() throws OutOfBoardException {
         //given
         Map<String, Object> playerInfo1 = Map.of(
-                "FieldStatus", FieldStatus.O,
+                "Field", Field.O,
                 "name", "Kasia",
                 "score", 0,
                 "first", false);
         Map<String, Object> playerInfo2 = Map.of(
-                "FieldStatus", FieldStatus.X,
+                "Field", Field.X,
                 "name", "Gosia",
                 "score", 0,
                 "first", true);
         UIExtended uiConsole = Mockito.mock(UIExtended.class);
         Mockito.when(uiConsole.readNumber()).thenReturn(1, 2, 3, 4, 5, 6, 7, 8, 9);
         PLAYER_CONTAINER.createPlayer(playerInfo1);
         PLAYER_CONTAINER.createPlayer(playerInfo2);
-        MatchController matchController = new MatchController(uiConsole, new Board(3), PLAYER_CONTAINER);
+        Match match = new Match(uiConsole, new Board(3, 3), PLAYER_CONTAINER);
 
         //when
```

Comment:
O na litość, czemu to nie jest jakaś metoda pomocnicza o dobrej nazwie, bym nie musiał ekranu przewijać by zrozumieć co właściwie jest testowane? Smutno mi Boże. :-1:

Reply:
Przeniosłem cześć danych jako stałe, wyodrębniłem metodę pomocniczą.